src/app/PersonOppslagHandler/LyttPåNyttFnrIReduxOgHentAllPersoninfo.tsx

```diff
@@ -33,6 +34,7 @@ function LyttPåNyttFnrIReduxOgHentAllPersoninfo() {
     useDispatchOnNewFnr(restResources.tråderOgMeldinger.actions.reset, fnr);
     useDispatchOnNewFnr(restResources.brukersVarsler.actions.reset, fnr);
     useFetchFeatureTogglesOnNewFnr();
+    useDispatchOnNewFnr(setDialogpanelTraad(undefined), fnr);
```

Comment:
Hva gjør setDialogpanelTraad(undefined)?

Reply:
den nullstiller redux-staten, men kan nok lage en egen action for å nullstille istedenfor å sende inn `undefined` som parameter, det er sikkert litt forvirrende